src/pods/movies/view-model/movie.ts

```diff
@@ -0,0 +1,7 @@
+export interface MovieEntity {
+    id: number;
+    title: string;
+    genre: string;
+    age_rating: number;
+    year: number;
+}
```

Comment:
Let's avoid creating a subfolder for this case, viewModel should be just 1 to 4 no more, so we can just place it in a file
viewmodel.ts